code/game/MapData/shuttles/nanotrasen_ranger.dm

```diff
@@ -458,5 +341,5 @@
 		/obj/item/disk/holodisk/lp/aid/guard = 1,
 	)
 
-	lootcount = 2
+	lootcount = 4
```

Comment:
So, how this var works, is that it's the amount of stuff one spawner spawns. I assume it's not intentional?